.agent/skills/memory/scripts/search.py

```
@@ -0,0 +1,65 @@
+        method="POST",
+    )
+    try:
+        with urllib.request.urlopen(req) as resp:  # noqa: S310
+            result = json.loads(resp.read())
+            for mem in result.get("results", []):
+                content = mem.get("content") or mem.get("memory") or ""
+                similarity = mem.get("score", 0)
+                print(f"[{similarity:.0%}] {content[:200]}")
+            if not result.get("results"):
+                print("No memories found.")
+    except Exception as e:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)